src/opencdms_api/climsoft_rbac_config.py

```diff
@@ -0,0 +1,183 @@
+required_role_lookup = {
+    "/climsoft/v1/climsoft-users": {
```

Comment:
Given this lookup is only used for Climsoft, would it make sense to remove the common part of the url: `/climsoft/v1`?

Otherwise we'll need to update each time the API version number changes.